.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Hmm. Unfortunae this is not blocked by our `.gitignore`.

Could you delete this file and add this line to our `.gitignore`: